components/announcement_bar/notify_admin_downgrade_delinquency_bar/index.tsx

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+
+import {FormattedMessage} from 'react-intl';
+import {useSelector, useDispatch} from 'react-redux';
+
+import {t} from 'utils/i18n';
+import {
+    AnnouncementBarTypes, CloudProductToSku, PaidFeatures, Preferences, TELEMETRY_CATEGORIES,
+} from 'utils/constants';
+
+import {GlobalState} from 'types/store';
+import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
+import {trackEvent} from 'actions/telemetry_actions';
+import {getSubscriptionProduct} from 'mattermost-redux/selectors/entities/cloud';
+import {NotifyStatus, useGetNotifyAdmin} from 'components/common/hooks/useGetNotifyAdmin';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+
+import AnnouncementBar from '../default_announcement_bar';
+import {useDelinquencySubscription} from 'components/common/hooks/useDelinquencySubscription';
+
+export const BannerPreferenceName = 'notify_upgrade_workspace_banner';
+
+const NotifyAdminDowngradeDelinquencyBar = () => {
+    const dispatch = useDispatch();
+    const getCategory = makeGetCategory();
+    const preferences = useSelector((state: GlobalState) =>
+        getCategory(state, Preferences.NOTIFY_ADMIN_REVOKE_DOWNGRADED_WORKSPACE),
+    );
+    const product = useSelector(getSubscriptionProduct);
+    const currentUser = useSelector((state: GlobalState) =>
+        getCurrentUser(state),
+    );
+    const {isDelinquencySubscriptionHigherThan90Days} = useDelinquencySubscription({
+        checkAdmin: false,
+    });
+
+    const {notifyAdmin, notifyStatus} = useGetNotifyAdmin({});
+
+    useEffect(() => {
+        if (notifyStatus === NotifyStatus.Success) {
+            dispatch(savePreferences(currentUser.id, [{
+                category: Preferences.NOTIFY_ADMIN_REVOKE_DOWNGRADED_WORKSPACE,
+                name: BannerPreferenceName,
+                user_id: currentUser.id,
+                value: 'adminNotified',
+            }]));
+        }
+    }, [currentUser, dispatch, notifyStatus]);
+
+    const shouldShowBanner = () => {
+        if (!preferences) {
+            return false;
+        }
+
+        if (preferences.some((pref) => pref.name === BannerPreferenceName)) {
+            return false;
+        }
+
+        if (!isDelinquencySubscriptionHigherThan90Days()) {
+            return false;
+        }
+
+        return true;
+    };
```

Comment:
Should we check here if there are not admins? I believe we only want to show this banner to end users